style.css

```diff
@@ -0,0 +1,60 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+}
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 50px;
+}
+.app {
+    width: 60%;
+}
+@media (max-width:800px){
+    .app {
+        width: 90%;
+    }
+}
+ h2 {
+    font-size: 40px;
+    text-align: center;
+    padding-top: 10px 0;
+}
+.app form {
+    display: flex;
+    width: 100%;
+}
+.app form input {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    font-size: 18px;
+    padding: 10px;
+    width: 70%;
+}
+.app form button {
+    background-color:lightseagreen;
+    color: white;
+    border-radius: 12px;
+    font-size: 25px;
+    font-weight: 100;
+    height: 42px;
+    width: 20%;
+    margin:0 0 0 auto;
+}
+.app ul {
+    list-style: none;
+    padding: 10px;
+}
+.app ul li {
+    cursor: pointer;
+    padding: 12px 10px 12px 40px;
+    background-color: whitesmoke;
+    border-bottom: 1px solid black;
+}
+ .app ul li i {
+    font-size: 30px;
+    float: right;
+    cursor: pointer;
+}
+
```

Comment:
Style items based on class names, rather than the element it self

Reply:
Added  class names and removed element names.